tracer.go

```diff
@@ -107,11 +107,11 @@ type StartSpanOptions struct {
 	// OperationName may be empty (and set later via Span.SetOperationName)
 	OperationName string
 
-	// Parent may specify Span instance that caused the new (child) Span to be
-	// created.
+	// Parent may specify the SpanContext that caused the new (child) Span to
+	// be created.
 	//
 	// If nil, start a "root" span (i.e., start a new trace).
-	Parent Span
+	Parent SpanContext
```

Comment:
nit: if we're going to change the Options struct, maybe we should go with the [functional options](https://commandcenter.blogspot.com.au/2014/01/self-referential-functions-and-design.html) instead. Doesn't need to be in the same PR, but I would vote for such change before bumping the API version. Only applies to Go, of course.